theme/brb/views/authentication/home.hbs

```diff
@@ -36,49 +36,48 @@
+                    <h4>{{$t "theme.brb.authentication.headline.welcomeToTheBrandenburgSchoolCloud"}}</h4>
+                    <p>{{$t "theme.brb.authentication.text.learningWithDigitalMedia"}}</p>
 
-					<h4Für die Schul-Cloud Brandenburg registrieren></h4>
-					<p>Eine einheitliche digitale Plattform für die ganze Schule bietet viele Vorteile. Die Registrierung für die Schul-Cloud Brandenburg wird daher durch die Schulleitung durchgeführt. Bitte wenden Sie sich an <a href="mailto:brandenburg@schul-cloud.org">brandenburg@schul-cloud.org</a>, wenn Sie Ihre Schule registrieren wollen.<br>
-						Anschließend erhalten alle Schüler:innen von ihrer Schule einen Registrierungslink oder Login-Daten für die erstmalige Anmeldung in der Schul-Cloud Brandenburg. Bei Fragen zur erstmaligen Anmeldung wenden sich Schüler:innen und Eltern bitte an den technischen Leiter ihrer Schule bzw. an die jeweilige Klassenlehrerin oder den jeweiligen Klassenlehrer.
+					<h4{{$t "theme.brb.authentication.headline.registerForBrandenburg"}}></h4>
+					<p>{{{$t "theme.brb.authentication.text.aUniformDigitalPlatform"}}}
 					</p>
 
-					<h4>Die HPI Schul-Cloud-Entwicklung zeichnet sich aus durch</h4>
+					<h4>{{$t "theme.brb.authentication.headline.theHPISchoolCloudDevelopment"}}</h4>
 					<ul>
-						<li>eine konsequente Nutzung von Open Source Lösungen,</li>
-						<li>eine durchgängig nutzerzentrierte Entwicklung,</li>
-						<li>einen strikt modularen Aufbau</li>
-						<li>die für ein Flächenland wichtige Funktion der schulübergreifenden Zusammenarbeit,</li>
-						<li>die EU-DSGVO/Datenschutz-Konformität durch Privacy by Design,</li>
-						<li>die Möglichkeit, durch Länderzusammenarbeit Synergien zu schaffen.</li>
+						<li>{{$t "theme.brb.authentication.text.consistentUseOfOpenSource"}}</li>
+						<li>{{$t "theme.brb.authentication.text.aConsistentlyUserCentred"}}</li>
+						<li>{{$t "theme.brb.authentication.text.aStrictlyModularStructure"}}</li>
+						<li>{{$t "theme.brb.authentication.text.theImportantFunction"}}</li>
+						<li>{{$t "theme.brb.authentication.text.dataProtectionConformity"}}</li>
+						<li>{{$t "theme.brb.authentication.text.theOpportunityToCreate"}}</li>
 					</ul>
 					<br>
 					<p>
-						<img src="https://schul-cloud-hpi.s3.hidrive.strato.com/brb/images/HPI_Schul_Cloud_Brandenburg_Schueler_2019.jpg" alt="Schüler sitzen zusammen und nutzen die Schul-Cloud Brandenburg auf ihren mobilen Endgeräten." />
-						Mehr Informationen: <a href="{{theme.documents.globalFiles.BeschreibungDerSchulCloud}}" target="_blank" rel="noopener">Kurzbeschreibung "Die HPI Schul-Cloud" (.pdf)</a>
+						<img src="https://schul-cloud-hpi.s3.hidrive.strato.com/brb/images/HPI_Schul_Cloud_Brandenburg_Schueler_2019.jpg" alt="{{$t "theme.brb.authentication.img_alt.schoolChildrenSitTogether"}}" />
+						{{$t "theme.brb.authentication.text.moreInformationHPISchoolCloud"}}</a>
```

Comment:
![image](https://user-images.githubusercontent.com/40116220/98006258-4f05bf80-1df2-11eb-8691-eefc327d556b.png)

Reply:
You should use here 3 {